server/service.go

```diff
@@ -355,3 +355,21 @@ func (m *BoostService) handleGetPayload(w http.ResponseWriter, req *http.Request
 		return
 	}
 }
+
+// CheckRelays sends a request to each one of the relays previously registered to get their status
+func (m *BoostService) CheckRelays() bool {
+	var status = false
+
+	for _, relay := range m.relays {
+		m.log.WithField("relay", relay).Info("Checking relay")
+
+		err := SendHTTPRequest(context.Background(), m.httpClient, http.MethodGet, relay.Address+pathStatus, nil, nil)
+		if err != nil {
+			m.log.WithError(err).WithField("relay", relay)
```

Comment:
```suggestion
			m.log.WithError(err).WithField("relay", relay).Error("relay check failed")
```
should this case also return false right away?

Reply:
I think we could define this behavior for `CheckRelays` until there is a need for more granular feedback about which relays work and which don't.

Reply:
Got it, added in b51ac76b2aed3fe6645384af27c2e6a42de2c128